### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mira1zu**

### CODE REVIEW

File: src/formatDate.js [hunk 1]
```diff
@@ -50,7 +50,36 @@
...
+    dateObj[fromFormat[i][0]] = dateSplit[i];
+  }
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    currentUnit = toFormat[i];
+    currentElement = dateObj[currentUnit[0]];
+
+    if (currentUnit.length < currentElement.length) {
+      currentElement = currentElement.slice(-2);
+    } else if (currentUnit.length > currentElement.length) {
```
Comment (jstmpelowycz) [inline src/formatDate.js:71]: if that's OK when `currentUnit.length === currentElement.length`, you can include it in one of the conditions, by this avoiding `else-if` statement, just `else` statement

File: src/formatDate.js [hunk 2]
```diff
@@ -50,7 +50,36 @@
...
+      if (currentElement < 30) {
+        currentElement = '20' + currentElement;
+      } else {
+        currentElement = '19' + currentElement;
+      }
```
Comment (illiarerun) [inline src/formatDate.js:76]: could've used ternary operator for it

**Review by illiarerun — APPROVED** [review]
Great job 🔥

File: src/formatDate.js (hunk 2 above)
Comment (jstmpelowycz) [inline src/formatDate.js:76]: nesting of conditions always a bad idea, so let's make some var like
```
const prefix = currentElement < 30 ? 20 : 19;
```
and then add before `currentElement`

**Review by jstmpelowycz — APPROVED** [review]
good job! take a look at what I've proposed

**Pull request closed without merging**